data_test.go

```diff
@@ -64,7 +65,7 @@ func TestParseData(t *testing.T) {
 			Payload: p[33:],
 		},
 	}
-	ds, err = parseData(ps, nil, pm)
+	ds, err = parseData(ps, nil, pm, esm)
```

Comment:
Could you add a test using `esm.setUnlocked()` like it was done with `pm.setUnlocked(uint16(256), uint16(1))` above?